### PR TITLE
Updates necessary for running jetson-xavier on L4T 32.2

### PR DESCRIPTION
**Pull request by acostach**

### CODE REVIEW

File: layers/meta-balena-jetson/conf/layer.conf [hunk 1]
```diff
@@ -19,6 +19,6 @@ BBMASK += "/meta-tegra/recipes-graphics/ "
 BBMASK += "/meta-tegra/external/openembedded-layer/recipes-support/opencv/ "
 
 LAYERSERIES_COMPAT_balena-jetson = "warrior"
-KERNEL_ARGS_jetson-xavier = "console=ttyTHS0,115200 OS=l4t fbcon=map:0"
+KERNEL_ARGS_jetson-xavier = "console=ttyTHS0,115200n8 OS=l4t fbcon=map:0 console=tty1 debug loglevel=7"
```
Comment (floion) [inline layers/meta-balena-jetson/conf/layer.conf]: Do we want the bootlog on tty1 in production images?

File: jetson-xavier.coffee [hunk 2]
```diff
@@ -2,9 +2,8 @@ deviceTypesCommon = require '@resin.io/device-types/common'
 { networkOptions, commonImg, instructions } = deviceTypesCommon
 
 BOARD_PREPARE  = 'Put the NVidia Jetson Xavier board in recovery mode'
-FLASH_SCRIPT_DOWNLOAD = 'Obtain the flashing script from: https://github.com/balena-os/balena-jetson-nano/blob/master/layers/meta-balena-jetson/recipes-scripts/flash_xavier.sh'
-FLASH_SCRIPT = 'Unzip BalenaOS image and execute: sudo ./flash_xavier.sh <path_to_balenaOS.img>'
-DONE_FLASHING  = 'Follow the instructions printed by the flashing script'
+FLASH_TOOL = 'Unzip BalenaOS image and use the Jetson Flash tool to flash the board'
```
Comment (floion) [inline jetson-xavier.coffee]: Let's also put a link here to where the flashing tool is.

File: layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra194-flash-dry_32.2.0.bb [hunk 3]
```diff
@@ -132,6 +132,7 @@ do_configure() {
 
     mkdir -p ${DEPLOY_DIR_IMAGE}/bootfiles
     cp ./cboot_t194.bin ${DEPLOY_DIR_IMAGE}/bootfiles/
+    echo "0" > ${DEPLOY_DIR_IMAGE}/bootfiles/none.bin
```
Comment (floion) [inline layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra194-flash-dry_32.2.0.bb]: What is this for?

Comment (floion) [reply layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra194-flash-dry_32.2.0.bb]: Is it the padding?

File: layers/meta-balena-jetson/conf/layer.conf (hunk 1 above)
Comment (acostach) [reply layers/meta-balena-jetson/conf/layer.conf]: No, we dont want that. Here we just override the machine.conf defined cmdline. Specifying console=null at the end for dev builds, like we do here: https://github.com/balena-os/balena-jetson/blob/e4fabce727ae9bb88d3cf4a27b7732477bf7fb61/layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra194-flash-dry_32.2.0.bb#L39 used to disable all the previous defined ones, but I'll check once again to be 100% sure nothing has changed.

File: jetson-xavier.coffee (hunk 2 above)
Comment (acostach) [reply jetson-xavier.coffee]: Thanks! Sure, will update

File: layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra194-flash-dry_32.2.0.bb (hunk 3 above)
Comment (acostach) [reply layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra194-flash-dry_32.2.0.bb]: Thanks @floion , I have updated the PR to address the requested changes.

**Pull request merged**